src/collections/blog/2022/2022-06-13-yash-gsoc-blogpost/index.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: "My Journey to GSoC @Layer5"
+subtitle: "A Rollercoaster experience"
+date: 2022-06-12 08:00:00 -0630
+author: Yash Kamboj
+thumbnail: ./Yash_Gsoc.png
+category: Community
+description: "Yash's journey to an GSoC @Layer5"
+tags:
+ - Community
+ - Projects
+ - News
+type: Blog
+published: true
+---
+
+import { BlogWrapper } from "../../Blog.style.js";
+
+
+
+<BlogWrapper>
+
+### My Early days with OpenSource
+
+I have been contributing to open source for some time now and have learned many new things. GSoC was where I learned about open source and then learned about many other open-source programs. In the starting Getting into one of the programs was my sole reason for contributing to open source, but as I started contributing, I found out how great open source is. I learned so many new things and got to work with so many amazing people. I enjoyed contributing so much that  I even forgot to apply to many programs, which was my sole reason to start contributing. I applied in GSoC 30mins before the deadline, and guess what, I got selected.
```

Comment:
"open source" is not hyphenated.

Reply:
Why is Getting uppercased?

Reply:
GSoC is where you learned open source, not Layer5?

Reply:
Let's not teach others that applying 30 minutes before the deadline is how they land internships with Layer5. They land internships by applying the counsel offered in the layer5.io/programs FAQs.

I hope that you didn't apply to internship programs only for resume-building, but because you believe in the projects here and want to see them succeed as much as you individually succeed.

I'm going to stop my review at this first paragraph and let you go back to the drawing board.

Reply:
Remove the extra whitespace in "contributing so much that  I even forgot"

Reply:
I don't understand this sentence - "In the starting Getting into one of the programs was my sole reason for contributing to open source, but as I started contributing, I found out how great open source is. " Please rephrase.

Reply:
The first part of this sentence is redundant with the second half of your opening sentence. "I learned so many new things and got to work with so many amazing people."